pulsar-client-go/pulsar/message.go

```diff
@@ -36,6 +36,9 @@ type ProducerMessage struct {
 
 	// Override the replication clusters for this message.
 	ReplicationClusters []string
+
+	// Set the sequence id to assign to the current message
+	ID int64
```

Comment:
I would use `SequenceID` here to avoid confusion with the "MessageId" concept which is different. eg. On a received message,  `Message.ID()` will give you the MessageId (assigned by brokers) rather than the sequence id (assigned by producer).

Reply:
At the same time, I think we should also expose a way to retrieve the sequence id on a received message as in `Message.SequenceID()` method.

Reply:
yes, `SequenceID` sounds good. about `Message.SequenceID()` method, i will provide in the next pull request.